jetty-documentation/src/main/asciidoc/administration/sessions/chapter.adoc

```diff
@@ -17,9 +17,21 @@
 [[session-management]]
 == Session Management
 
-include::setting-session-characteristics.adoc[]
-include::using-persistent-sessions.adoc[]
-include::session-clustering-jdbc.adoc[]
-include::session-clustering-mongodb.adoc[]
-include::session-clustering-infinispan.adoc[]
-include::session-clustering-gcloud-datastore.adoc[]
\ No newline at end of file
+Sessions are a concept within the Servlet api which allow requests to store and retrieve information across the time a user spends in an application.
+Choosing the correct session manager implementation is an important consideration for every application as each can fit and perform optimally in different situations.
+If you need a simple in-memory session manager that can persist to disk then the File Session Manager can be a good place to start.
```

Comment:
I wouldn't refer to it as the File Session Manager, and in general I would try to avoid the phrase "session manager" altogether:  maybe try rephrasing as "session management" instead.